.gitignore

```diff
@@ -1,4 +1,122 @@
-build
-dist
-certs
-node_modules
+
```

Comment:
I'm not sure including globals gitignore rules in the repo is good practice.

See https://docs.microsoft.com/en-us/azure/devops/repos/git/ignore-files?view=azure-devops&tabs=visual-studio#ignore-files-only-on-your-system